src/App.js

```diff
@@ -7,23 +7,37 @@ import EnterName from "./name/EnterName";
 
 export const App = ({ config: { state, indicators } }) => {
   const [petName, setPetName] = useState(state.name);
-  if (indicators.age.maxAge > 20) {
-    return (
-      <div>
-        <p className={style.error}>ERROR</p>
-      </div>
-    );
-  } else {
-    if (petName) {
+  if (indicators.age.maxAge) {
+    if (indicators.age.maxAge > 20) {
+      console.log("REACT_APP_MAX_AGE is greater than 20");
```

Comment:
better use `console.error` if it's an error. `console.log` is for information

Reply:
done